privateSBOMExchange/tests/test_cpabe.py

```diff
@@ -0,0 +1,96 @@
+import configparser
+import copy
+import sys
+import os
+import configparser
+import filecmp
+from difflib import unified_diff
+from lib4sbom.parser import SBOMParser
+
+from ..src.petra.lib.models import *
```

Comment:
Is the installation of the Petra package not working? This is more of a stylistic comment, using our code by installing it via `pip install -e` and importing it like any other package is preferable over relative path imports.

Reply:
It didn't work for me. But I can push without the relative paths

Reply:
It works for me now